stdlib/Serialization/src/Serialization.jl

```diff
@@ -1191,6 +1191,9 @@ function deserialize(s::AbstractSerializer, ::Type{CodeInfo})
     end
     ci.propagate_inbounds = deserialize(s)
     ci.pure = deserialize(s)
+    if format_version(s) >= 19
```

Comment:
I think you need to bump the format version to 20. Adding the field changes the serialization.

Reply:
Done